src/Library/demos/Toasts/main.vala

```diff
@@ -0,0 +1,37 @@
+public void main () {
+  var overlay = workbench.builder.get_object ("overlay") as Adw.ToastOverlay;
+
+  var button_simple = workbench.builder.get_object ("button_simple") as Gtk.Button;
+  button_simple.clicked.connect (() => {
+    var toast = new Adw.Toast ("Toasts are delicious!") {
+      timeout = 1
+    };
+
+    toast.dismissed.connect (() => {
+      button_simple.set_sensitive (true);
+    });
+    overlay.add_toast (toast);
+    button_simple.set_sensitive (false);
+  });
+
+  var button_advanced = workbench.builder.get_object("button_advanced") as Gtk.Button;
+  button_advanced.clicked.connect(() => {
+    var message_id = "42";
```

Comment:
please use ```int``` instead of ```var``` here.

Reply:
This is actually a `string` although I could change it to an `int` and then use an `int.parse ()` on line 42, but that seems a tad redundant. Ok to just change it to `string` on line 19 here?

Reply:
Yeah. I lookedntonquickly over it :?/)